bmi_calculator/bmi_calculator.py

```diff
@@ -0,0 +1,129 @@
+import sys
+
+from PyQt6.QtWidgets import (
+    QApplication,
+    QDialog,
+    QWidget,
+    QLabel,
+    QRadioButton,
+    QHBoxLayout,
+    QComboBox,
+    QButtonGroup,
+    QDialogButtonBox,
+    QFormLayout,
+    QLineEdit,
+    QVBoxLayout,
+    QPushButton,
+)
+class MyGUIWindow(QDialog):
+    def __init__(self):
+        super().__init__(parent=None)
+        self.setStyleSheet("background-color: lightblue;")
+        self.setWindowTitle("BMI Calculator")
+        self.setGeometry(400,200,400,300)
+
+        dialoglayout = QVBoxLayout()
+        self.setLayout(dialoglayout)
+
+        formlayout = QFormLayout()
+        #Radio button groups
+        self.rbtn_group_gender = QButtonGroup()
+        self.rbtn_group_height = QButtonGroup()
+        self.rbtn_group_weight = QButtonGroup()
+
+        gender = QLabel("Select the Gender:")
+        dialoglayout.addWidget(gender)
+
+        hor_layout1 = QHBoxLayout()
+        dialoglayout.addLayout(hor_layout1)
+
+        self.r1_rbtn_male = QRadioButton("Male")
+        self.r2_rbtn_female = QRadioButton("Female")
+        hor_layout1.addWidget(self.r1_rbtn_male)
+        hor_layout1.addWidget(self.r2_rbtn_female)
+
+        self.rbtn_group_gender.addButton(self.r1_rbtn_male)
+        self.rbtn_group_gender.addButton(self.r2_rbtn_female)
+
+
+        height = QLabel("Enter the Height:")
+        dialoglayout.addWidget(height)
+
+        hor_layout2 = QHBoxLayout()
+        dialoglayout.addLayout(hor_layout2)
+
+        self.r2_rbtn_cms = QRadioButton("Cms")
+        self.r2_rbtn_inches = QRadioButton("Inches")
+        hor_layout2.addWidget(self.r2_rbtn_cms)
+        hor_layout2.addWidget(self.r2_rbtn_inches)
+
+        self.rbtn_group_height.addButton(self.r2_rbtn_cms)
+        self.rbtn_group_height.addButton(self.r2_rbtn_inches)
+
+        self.combo_box = QComboBox()
+        self.combo_box.setStyleSheet("border: 1px solid black;")
+        self.combo_box.setStyleSheet("background-color: lightgrey;")
+        for i in range(45,280):
+            self.combo_box.addItem(str(i))
+        dialoglayout.addWidget(self.combo_box)
+
+
+        self.weight = QLineEdit()
```

Comment:
insted of `self.weight ` --> `  self.text_filed_weight`, try to updated the UI object names with clear  differentiation